304-known-implementations.md

```diff
@@ -0,0 +1,123 @@
+---
+title: CNAB Security: Known implementations
+weight: 304
+---
+
+The security implementation prescribes the structure and format that trust metadata SHOULD follow, but _not_ which specific implementations should be used. This document presents known implementations of the CNAB security specification, and as more projects implement the specification, they will be added here.
+
+### [Signy][signy]
+
+[Signy][signy] is a project built to validate the feasability of integrating the [CNAB registry][cnab-reg] and CNAB security specifications by demonstrating the complete sign-push-verify workflow.
+
+The current document is based on [v0.0.3][signy-release] of Signy, which:
```

Comment:
Heads up that the link to that release of signy is broken because there is a draft release https://github.com/cnabio/signy/releases/tag/untagged-00ed798ca223bbe46852 but no tag yet. Once you cut the release the link will work. Are you planning on releasing these (this doc) and that version of signy together?

Reply:
Thanks for pointing it out, just released the tag.